### DISCUSSION TITLE
Leveraging Discussions as a content source

### DISCUSSION THREAD

**Discussion by zarahzachz** [CMSgov/design-system#1822] [Ideas]
We've been using Discussions for about a week now and I have some ideas on ways we could leverage the content created in Discussions more.

Initial ideas include:
1. Moving RFCs to Discussions
2. Using Github's GraphQL API to post content to our documentation site
3. Goal-setting using community insights

## Moving RFCs to Discussions

As it stands, RFCs live within a directory in our codebase, are written by engineers, and discussed within PRs.

This way of working presents several limitations:
- RFCs can only be accessed by those who know where they are and can comfortably navigate a codebase
- RFCs are usually only reviewed by engineers on the team, so feedback is limited to a select few
- Contribution is limited to those who can comfortably write Markdown and navigate a codebase
- Discussion is somewhat ephemeral, happening within an RFC PR and "disappearing" once that PR is closed*

* This content still exists, but it loses discoverability upon closure.

Moving our RFC process to GH Discussions...
- Can be more easily accessed by the public and can be linked to in documentation
- Facilitates more discussion with non-engineers, maybe even with those outside our team
- Widens the pool on who can contribute an RFC (engineering, design, product)
- Discussion around the RFC remains intact and is referance-able

## Using Github's GraphQL API to post content to our documentation site

[Github has a GraphQL API](https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions) that can be used to get data from Discussions. What kind of data we want and how we want it displayed is still up in the air, but one idea could be to use it to post RFCs, release notes, and other important announcements to our documentation site as a way to over-communicate upcoming design system information in a very public way.

The API can target specific Discussion categories, labels, pinned status, etc. Permissions for setting these attributes on a Discussion can be limited to those with specific admin privileges.

Another way we could leverage this API is by piloting a newsletter-type campaign where a post can be written in Discussions detailing the latest from both design and development, and be served on the documentation site for extra awareness. 

Pulling content from Discussions into our documentation site makes the site even more of a single source of truth to our users.

## Goal-setting using community insights

Discussions offers a [community insights](https://github.com/CMSgov/design-system/graphs/community) feature, which is a dashboard of user engagement. This data can help us craft SMART goals during PI planning but having metrics to work against.

**Comment by pwolfert** [comment]
Great ideas! I'm going to work on moving our existing RFCs to discussions
